civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -0,0 +1,22 @@
+.Position-entry {
+  background-color: white;
```

Comment:
Not a breaking change since they're both `#ffffff` at the moment but if time permits I'd change that to the background variable instead of the text one in case we do re-theme DL.